syntax = "proto3";

package infrastructure.proto.user.v1;

import "app/infrastructure/proto/v1/user/model.proto";

message GetUserRequest {
  string id = 1;
}

message GetUserResponse {
  User user = 1;
}
